### PR TITLE
{bio}[foss/2019a] ANGEL v3.0 w/ Python 3.7.2

### PR DESCRIPTION
**Pull request by manifestoso**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by robqiao** [comment]
Do we have a mechanism for step control on the `sanity_check` in the config file? 
The issue here was that `PythonPackage` defines `python -c "import <module>"`, can we bypass this step instead of reconstructing an easy_block? Temp solution here was to skip the `sanitycheck` step altogether.

**Comment by boegel** [comment]
> Do we have a mechanism for step control on the `sanity_check` in the config file?
> The issue here was that `PythonPackage` defines a `python -c "import <module>"` step, can we bypass this step instead of reconstructing an easy_block? Temp solution here was to skip the `sanitycheck` step altogether.

You can use this to skip the `import` check, which is way better than skipping the whole sanity check:

```python
options = {'modulename': False}

File: easybuild/easyconfigs/a/ANGEL/ANGEL-3.0-foss-2019a-Python-3.7.2.eb [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+local_bin_files = [
+    'dumb_predict.py',
+    'angel_make_training_set.py',
+    'angel_train.py',
+    'angel_predict.py',
+    'angel_compare_files.py'
+]
+
+options = {'modulename': False}
+
```
Comment (smoors) [inline easybuild/easyconfigs/a/ANGEL/ANGEL-3.0-foss-2019a-Python-3.7.2.eb:46]: @robqiao please add here:
```
sanity_pip_check = True
```

(@boegel this is not checked by the tests?)

**Review by smoors — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/a/ANGEL/ANGEL-3.0-foss-2019a-Python-3.7.2.eb (hunk 1 above)
Comment (verdurin) [reply easybuild/easyconfigs/a/ANGEL/ANGEL-3.0-foss-2019a-Python-3.7.2.eb:46]: @robqiao any update on the request from @smoors ?

Comment (boegel) [reply easybuild/easyconfigs/a/ANGEL/ANGEL-3.0-foss-2019a-Python-3.7.2.eb:46]: https://github.com/DeepThoughtHPC/easybuild-easyconfigs/pull/8

Comment (boegel) [reply easybuild/easyconfigs/a/ANGEL/ANGEL-3.0-foss-2019a-Python-3.7.2.eb:46]: @robqiao Can you be a bit more specific? Which dep versions will `pip` overwrite when enabling `use_pip`?

**Comment by smoors** [comment]
@boegelbot: please test @ generoso

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node300.hydra.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz (skylake_avx512), Python 2.7.5
See https://gist.github.com/39c23167af5ab50c430e6e99e633e012 for a full test report.

**Review by smoors — APPROVED** [review]
lgtm

**Comment by smoors** [comment]
Going in, thanks @manifestoso!

**Pull request merged**